src/Maker/MakeEvent.php

```diff
@@ -0,0 +1,257 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\EventRegistry;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Bundle\MakerBundle\Util\UseStatementGenerator;
+use Symfony\Bundle\MakerBundle\Validator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\EventDispatcher\Attribute\AsEventListener;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+
+/**
+ * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ * @author Ryan Weaver <weaverryan@gmail.com>
+ * @author Steven Renaux <steven.renaux8000@gmail.com>
+ */
+final class MakeEvent extends AbstractMaker
+{
+    private const ALL_TYPES = ['Subscriber', 'Listener'];
+    private bool $isSubscriber = false;
+
+    public function __construct(private readonly EventRegistry $eventRegistry)
+    {
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:event';
+    }
+
+    /**
+     * @deprecated remove this method when removing make:subscriber
+     */
+    public static function getCommandAlias(): string
+    {
+        return 'make:subscriber';
+    }
```

Comment:
Wait, will this mean that `make:subscriber` will now call THIS maker? If so, let's not do this. Instead, let's keep `make:subscriber` as it is, but deprecate it.

Reply:
I made a deprecation and followed the tricks as it was done for `make:unit-test` and `make:functional-test`.
When `make:subscriber` is called it run my maker and alert the user.

https://github.com/symfony/maker-bundle/blob/ab025cd05b93e9b3955e7416cefede5e87fb9deb/src/Maker/MakeTest.php#L60

Reply:
Hmm, ok fair enough. We are free to change what these commands due between minor releases. So let's give them the better experience.